src/ui/FormButton/FormButton.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import styles from './FormButton.module.css';
+
+interface FormButtonInterface extends React.ComponentProps<'button'> {
+    isActive?: boolean;
+}
+
+function FormButton({ className, children, isActive, ...props }: FormButtonInterface) {
+    let classes = styles.formButton;
+
+    if (className) {
+        classes += ` ${className}`;
+    }
+
+    if (isActive) {
+        classes += ` ${styles.active}`;
+    }
```

Comment:
create a function for merge classes, and use it all over the app
https://www.felixmokross.dev/blog/merging-css-classes